vendor/templates/vendor/manage/offer.html

```diff
@@ -50,18 +398,41 @@ <h4>{% trans 'Price' %}</h4>
             <div class='card-body'>
                 <div class='dropdown'>
                     <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton"
-                        data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                        {% trans 'Add Customer' %}
-                    </button>
-                    <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-                        {% for customer_profile in customers_who_dont_own %}
-                        <a class="dropdown-item" href="{% url 'vendor_admin:manager-profile-add-offer' customer_profile.uuid offer.uuid %}">{{customer_profile}}</a>
-                        {% endfor %}
-                    </div>
+                    data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    {% trans 'Add Customer' %}
+                </button>
+                <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                    {% for customer_profile in customers_who_dont_own %}
+                    <a class="dropdown-item"
+                    href="{% url 'vendor_admin:manager-profile-add-offer' customer_profile.uuid form.instance.uuid %}">{{customer_profile}}</a>
+                    {% endfor %}
                 </div>
             </div>
         </div>
     </div>
 </div>
+</div>
+{% endcomment %}
+{% endblock %}
 
+{% block extra_js %}
+<script src="https://code.jquery.com/jquery-1.12.4.js"></script>
```

Comment:
Are these external libraries loaded elsewhere in the project?  They should only be included once.

Reply:
You are right, moved it to the base file and replaced it with min instead of slim. Given that slim those not have the required vars for jquery-ui